src/Guardrails-Common/GR-Common.psm1

```diff
@@ -737,22 +738,24 @@ function Parse-BlobContent {
     }
 
     # Split content into lines
-    $lines = $blobContent -split "`r`n|`n"
+    $lines = $blobContent -split "`r`n|`n|,|;|,\s|;\s"
 
     $filteredLines = $lines | Where-Object { $_ -match '\S' -and $_ -like "*@*" } | ForEach-Object { $_ -replace '\s' }
 
     # Initialize an empty array
     $globalAdminUPNs = @()
 
-    # Check each line, remove the hyphen, and add to array
+    # Check each line, remove the hyphen (if any), and add to array
     foreach ($line in $filteredLines) {
         if ($line.StartsWith("-")) {
             # Remove the leading hyphen and any potential whitespace after it
-            $trimmedLine = $line.Substring(1).Trim()
-            $globalAdminUPNs += $trimmedLine
-        } else {
-            throw "Invalid format found: $line"
+            $trimmedLine = $line.Substring(1)
```

Comment:
I think .Trim() should still be there as it removes any whitespace added after UPN

for example
user@example1.com   ;user2@example.com; user3@example.com
Above will result in $trimmedLine evaulating forch UPN to 
user@example1.com[:SPACES] ----> Here Trim would have removed the trailing whitespace
user2@example.com
user3@example.com